msgid ""
msgstr ""
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-07-22 19:19+0200\n"
"PO-Revision-Date: \n"
"Last-Translator: Pedro_Newbie <backupsuite@outlook.com>\n"
"Language-Team: \n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Poedit-SourceCharset: utf-8\n"
"X-Generator: Poedit 2.1.1\n"

#: ../plugin.py:441
msgid ""
"\n"
"The found files:"
msgstr ""
"\n"
"Gefundene Dateien:"

#: ../plugin.py:358
msgid ""
"\n"
"You are using dual multiboot!"
msgstr ""

#: ../plugin.py:447
#, python-format
msgid "  %s (maybe error)"
msgstr " %s (vielleicht fehlerhaft)"

#: ../plugin.py:444
#, python-format
msgid "  %s (maybe ok)"
msgstr " %s (vielleicht ok)"

#: ../message.py:180
msgid " MB available space"
msgstr " MB Speicher frei"

#: ../message.py:185
msgid " MB needed space"
msgstr " MB Speicher benötigt"

#: ../plugin.py:450
msgid " nothing!"
msgstr "Nichts!"

#: ../message.py:45
msgid " size to be backed up: "
msgstr " Größe zu sichern: "

#: ../message.py:175
msgid " to make a back-up!"
msgstr " um ein Backup zu erstellen!"

#: ../message.py:169
msgid "* * * WARNING * * *"
msgstr "* * * WARNUNG * * *"

#: ../plugin.py:480
msgid "A recording is currently running. Please stop the recording before trying to start a flashing."
msgstr "Aktuell läuft eine Aufnahme. Bitte die Aufnahme stoppen bevor versucht wird einen Flashvorgang zu starten."

#: ../message.py:70
msgid "Additional backup -> "
msgstr "Zusätzliche Sicherung -> "

#: ../message.py:35
msgid "BACK-UP TOOL, FOR MAKING A COMPLETE BACK-UP"
msgstr "BACK-UP TOOL, UM EIN VOLLSTÄNDIGES SYSTEM-BACK-UP ZU ERSTELLEN"

#: ../plugin.py:132
msgid "Backup > HDD"
msgstr "Backup > HDD"

#: ../plugin.py:130
#, fuzzy
msgid "Backup > MMC"
msgstr "Backup > HDD"

#: ../plugin.py:133
msgid "Backup > USB"
msgstr "Backup > USB"

#: ../plugin.py:584 ../plugin.py:591
msgid "Backup and restore your image"
msgstr ""

#: ../message.py:149
msgid "Backup done with: "
msgstr "Backup erstellt mit: "

#: ../message.py:106
msgid "Backup finished and copied to your USB-flashdrive."
msgstr "Das Backup ist fertig und wurde auf den USB-Stick kopiert."

#: ../plugin.py:338 ../plugin.py:541
#, fuzzy
msgid "Backup info"
msgstr "BackupSuite"

#: ../plugin.py:583 ../plugin.py:590
msgid "BackupSuite"
msgstr "BackupSuite"

#: ../plugin.py:131 ../plugin.py:233 ../plugin.py:262
msgid "Close"
msgstr ""

#: ../message.py:65
msgid "Create: kerneldump"
msgstr "Erstelle: Kerneldump"

#: ../message.py:60
msgid "Create: root.ubifs"
msgstr "Erstelle: root.ubifs"

#: ../plugin.py:339 ../plugin.py:570
msgid "Delete"
msgstr ""

#: ../plugin.py:567
#, fuzzy
msgid "Delete backup"
msgstr "Backup flashen"

#: ../plugin.py:540
#, python-format
msgid "Do you really want to unpack %s ?"
msgstr "Möchten sie %s wirklich entpacken ?"

#: ../plugin.py:153
msgid ""
"Do you want to make a back-up on USB?\n"
"\n"
"This only takes a few minutes depending on the used filesystem and is fully automatic.\n"
"\n"
"Make sure you first insert an USB flash drive before you select Yes."
msgstr ""
"Möchten sie ein Backup auf USB erstellen?\n"
"\n"
"Das dauert abhängig vom verwendeten Dateisystem nur wenige Minuten und geht automatisch.\n"
"\n"
"Stellen Sie sicher, dass ein USB-Stick angeschlossen ist, bevor Sie Ja wählen."

#: ../plugin.py:150
msgid ""
"Do you want to make an USB-back-up image on HDD? \n"
"\n"
"This only takes a few minutes and is fully automatic.\n"
msgstr ""
"Möchten sie ein USB-Backup auf HDD erstellen? \n"
"\n"
"Das dauert nur wenige Minuten und geht automatisch.\n"

#: ../plugin.py:156
#, fuzzy
msgid ""
"Do you want to make an USB-back-up image on MMC? \n"
"\n"
"This only takes a few minutes and is fully automatic.\n"
msgstr ""
"Möchten sie ein USB-Backup auf HDD erstellen? \n"
"\n"
"Das dauert nur wenige Minuten und geht automatisch.\n"

#: ../plugin.py:470
msgid "Exit"
msgstr "Beenden"

#: ../plugin.py:484
msgid "Flashing: "
msgstr "Flash läuft..."

#: ../plugin.py:369
msgid "For flashing your receiver files are needed:\n"
msgstr "Benötigte Dateien um zu flashen:\n"

# #: message22
#: ../message.py:129
msgid "Full back-up direct to USB"
msgstr "Voll-Backup direkt auf USB"

#: ../plugin.py:194
msgid "Full back-up on HDD"
msgstr "Voll-Backup auf HDD"

#: ../plugin.py:208
#, fuzzy
msgid "Full back-up on MMC"
msgstr "Voll-Backup auf HDD"

#: ../plugin.py:201
msgid "Full back-up to USB"
msgstr "Voll-Backup auf USB"

#: ../message.py:235
#, fuzzy
msgid "Full back-up to the MultiMediaCard"
msgstr "Voll-Backup auf Festplatte"

#: ../message.py:111
msgid "Full back-up to the harddisk"
msgstr "Voll-Backup auf Festplatte"

#: ../message.py:123
msgid "If you place an USB-stick containing this file then the back-up will be automatically made onto the USB-stick and can be used to restore the current image if necessary."
msgstr "Steckt man einen USB-Stick, der diese Datei beinhaltet, an die Box, wird das Backup automatisch auf diesen USB-Stick erstellt und kann bei Bedarf zur Wiederherstellung des aktuellen Images verwendet werden."

#: ../message.py:90
msgid "Image creation FAILED!"
msgstr "Image-Erstellung fehlgeschlagen!"

#: ../message.py:225
msgid "Installed packages contained in this backup:"
msgstr ""

#: ../message.py:154
msgid "KB per second"
msgstr "KB pro Sekunde"

#: ../plugin.py:127
msgid "Make a backup or restore a backup"
msgstr "Backup erstellen oder wiederherstellen"

#: ../message.py:159
msgid "Most likely this back-up can't be restored because of it's size, it's simply too big to restore. This is a limitation of the bootloader not of the back-up or the BackupSuite."
msgstr "Dieses Backup kann höchstwahrscheinlich nicht wieder eingespielt werden, es ist schlicht zu gross um es wieder herzustellen. Dies ist eine Beschränkung des Bootloaders, nicht des Backups oder der BackupSuite."

#: ../plugin.py:45
msgid "NAS"
msgstr ""

#: ../message.py:230
msgid "NFI Image created in: "
msgstr ""

#: ../plugin.py:516
msgid "NOT found files for flashing!\n"
msgstr "Keine Dateien zum Flashen gefunden !\n"

#: ../message.py:220
msgid "No additional USB-stick found to copy an extra backup"
msgstr ""

#: ../plugin.py:173 ../message.py:30
msgid "No supported receiver found!"
msgstr "Kein unterstützter Receiver gefunden!"

#: ../message.py:170
msgid "Not enough free space on "
msgstr "Nicht genügend Speicher frei auf "

#: ../plugin.py:458 ../plugin.py:499
msgid "Only kernel"
msgstr "Nur Kernel"

#: ../plugin.py:457 ../plugin.py:496
msgid "Only root"
msgstr "Nur root"

#: ../message.py:118
msgid "PLEASE READ THIS:"
msgstr "BITTE LESEN SIE DIESES:"

#: ../message.py:40
msgid "Please be patient, a backup will now be made, this will take about: "
msgstr "Bitte etwas Geduld, ein Backup wird erstellt, das dauert ungefähr: "

#: ../message.py:85
msgid "Please check the manual of the receiver on how to restore the image."
msgstr "Bitte lesen Sie im Handbuch des Receivers, wie man ein Image wieder herstellt."

#: ../plugin.py:234
msgid "Please wait"
msgstr ""

#: ../plugin.py:528
msgid "Please: DO NOT reboot your STB and turn off the power.\n"
msgstr "Bitte die Box keinesfalls neustarten bzw. in Power OFF versetzen !\n"

#: ../message.py:215
msgid "Probably you are trying to make the back-up in flash memory"
msgstr ""

#: ../plugin.py:134
msgid "Restore backup"
msgstr "Backup flashen"

#: ../plugin.py:336 ../plugin.py:353 ../plugin.py:362
msgid "Run flash"
msgstr "Starte Flash"

#: ../plugin.py:464 ../plugin.py:511
#, fuzzy
msgid "Second partition (only kernel)"
msgstr "Standard (root und Kernel)"

#: ../plugin.py:463 ../plugin.py:508
msgid "Second partition (only root)"
msgstr ""

#: ../plugin.py:462 ../plugin.py:505
#, fuzzy
msgid "Second partition (root and kernel)"
msgstr "Standard (root und Kernel)"

#: ../plugin.py:368
msgid "Select parameter for start flash!\n"
msgstr "Wähle die Art zu flashen !\n"

#: ../plugin.py:261
msgid "Select the folder with backup"
msgstr "Wähle den Ordner mit dem Backup"

#: ../plugin.py:521
msgid "Show only found image and mtd partitions.\n"
msgstr "Zeige verfügbare Images und MTD Partitionen.\n"

#: ../plugin.py:455 ../plugin.py:487
msgid "Simulate (no write)"
msgstr "Simulation (kein Schreibvorgang)"

#: ../plugin.py:461 ../plugin.py:502
#, fuzzy
msgid "Simulate second partition (no write)"
msgstr "Simulation (kein Schreibvorgang)"

#: ../message.py:55
msgid "Some information about the task"
msgstr "Einige Informationen über die Aufgabe"

#: ../plugin.py:456 ../plugin.py:490
msgid "Standard (root and kernel)"
msgstr "Standard (root und Kernel)"

#: ../message.py:134
msgid "The content of the folder is:"
msgstr "Der Inhalt des Ordners ist:"

#: ../plugin.py:529
msgid "The image or kernel will be flashing and auto booted in few minutes.\n"
msgstr "Das Image oder der Kernel wird geflasht - Box startet in wenigen Minuten neu..\n"

#: ../message.py:190
msgid "The program will abort, please try another medium with more free space to create your back-up."
msgstr "Das Programm wird abgebrochen, bitte ein anderes Medium mit mehr freien Speicher wählen um ihr Backup zu erstellen."

#: ../message.py:124
msgid "The program will exit now."
msgstr "Das Programm wird jetzt beendet."

#: ../message.py:164
msgid "There COULD be a problem with restoring this back-up because the size of the back-up comes close to the maximum size. This is a limitation of the bootloader not of the back-up or the BackupSuite."
msgstr "Bei der Wiederherstellung dieses Backup könnte es zu einem Problem kommen da dieses nahe an der zulässigen Grösse liegt. Dies ist eine Beschränkung des Bootloaders, nicht des Backups oder der BackupSuite."

#: ../message.py:116
msgid "There is NO valid USB-stick found, so I've got nothing to do."
msgstr "Es wurde kein USB-Stick gefunden, Abbruch...."

#: ../message.py:100
msgid "There is a valid USB-flashdrive detected in one of the USB-ports, therefore an extra copy of the back-up image will now be copied to that USB-flashdrive."
msgstr "Es wurde ein USB-Stick an einem der USB-Anschlüsse gefunden, es wird nun eine Extra-Kopie des Backup-Images auf den USB-Stick kopiert."

#: ../message.py:101
msgid "This only takes about 20 seconds....."
msgstr "Das dauert nur ungefähr 20 Sekunden....."

#: ../message.py:139
msgid "Time required for this process: "
msgstr "Die benötigte Zeit für dieses Backup: "

#: ../message.py:119
msgid "To back-up directly to the USB-stick, the USB-stick MUST contain a file with the name:"
msgstr "Um ein Backup auf einem USB-Stick erstellen zu können, muss auf dem USB-Stick eine Datei mit dem folgenden Namen vorhanden sein:"

#: ../message.py:75
msgid "USB Image created in: "
msgstr "USB-Image erstellt in: "

#: ../plugin.py:334 ../plugin.py:537
msgid "Unzip"
msgstr "Entpacken"

#: ../message.py:200
msgid "Using your own custom lookuptable.txt from the folder /etc"
msgstr ""

#: ../message.py:205
msgid "Version unknown, probably not installed the right way."
msgstr ""

#: ../plugin.py:359
msgid ""
"Warning!\n"
"Use at your own risk! Make always a backup before use!\n"
"Don't use it if you use multiple ubi volumes in ubi layer!"
msgstr ""
"Warnung !\n"
"!!! Verwendung auf eigene Gefahr !!! Zuvor immer ein Backup erstellen!\n"
"Nicht anwenden bei multiple ubi volumes in ubi layer"

#: ../plugin.py:232
msgid "What is new since the last release?"
msgstr "Was ist neu seit der letzten Version?"

#: ../plugin.py:571
#, python-format
msgid ""
"You are about to delete this backup:\n"
"\n"
"%s\n"
"Continue?"
msgstr ""

#: ../plugin.py:527
msgid "Your STB will freeze during the flashing process.\n"
msgstr "Während dem Flashvorgang ist die Box unbedienbar.\n"

#: ../message.py:80
msgid "and there is made an extra copy in: "
msgstr "Eine Extra-Kopie liegt in: "

#: ../message.py:95
msgid "available "
msgstr "verfügbar "

#: ../message.py:120
msgid "backupstick or"
msgstr "backupstick oder"

#: ../message.py:121
msgid "backupstick.txt"
msgstr "backupstick.txt"

#: ../plugin.py:266 ../plugin.py:328
#, python-format
msgid "current:  %s"
msgstr "Aktuell:  %s"

#: ../message.py:195
msgid "is not executable..."
msgstr ""

#: ../message.py:144
msgid "minutes"
msgstr "Minuten"

#: ../message.py:50
msgid "not found, the backup process will be aborted!"
msgstr "nicht gefunden, der Backup-Vorgang wird abgebrochen!"

#: ../message.py:210
msgid "not installed yet, now installing"
msgstr ""

#: ../plugin.py:526
msgid "ofgwrite will stop enigma2 now to run the flash.\n"
msgstr "Das Programm ofgwrite stoppt jetzt Enigma2 um den Flash zu starten.\n"
